beta-component-promotion/README.md

```diff
@@ -23,21 +23,16 @@ as well as relevant examples in the documentation are all labeled as beta.
 
 | *Component currently in beta*          | inital @patternfly/patternfly milestone | inital @patternfly/patternfly-react milestone |
 |----------------------------------------|-----------------------------------------|-----------------------------------------------|
-| Button: with count                     | 2022.11                                 | 2022.12                                       |
-| DataList: draggable                    | 2021.12                                 | 2020.13                                       |
-| Drag and drop                          | 2021.13                                 | 2021.14                                       |
```

Comment:
Drag/drop is still beta on org staging and [in core](https://github.com/patternfly/patternfly/blob/f75001a77a5045d018059f8f9447e53e64a7fee7/src/patternfly/components/DragDrop/examples/DragDrop.md?plain=1#L3), should take out `beta: true` in core?

Reply:
yes 👍🏻